modules/async-icq/testing/demo-simapp/README.md

```diff
@@ -0,0 +1,100 @@
+# interchainquerydemo
+**ICQ Demo** 
+
+## Disclaimer
+
+The following repository and [`x/interquery`](./x/interquery/) module serves as an example and is used to exercise the functionality of Interchain Accounts end-to-end for development purposes only.
```

Comment:
I know this was incorrect in the original repo, `Interchain Accounts` should be replaced with `async-icq`

Reply:
053a49f9ef48e70b3f49267db7cf95d8935bf620